src/scancode/cli.py

```diff
@@ -664,7 +664,10 @@ def save_results(files_count, scanned_files, format, input, output_file):
         meta['files_count'] = files_count
         # TODO: add scanning options to meta
         meta['files'] = scanned_files
-        output_file.write(unicode(json.dumps(meta, indent=2 * ' ', iterable_as_array=True, encoding='utf-8')))
+        if format == 'json-pp':
+            output_file.write(unicode(json.dumps(meta, indent=2 * ' ', iterable_as_array=True, encoding='utf-8')))
+        else:
+            output_file.write(unicode(json.dumps(meta, iterable_as_array=True, encoding='utf-8')))
```

Comment:
Could there be a way to have an even more compact JSON? OTH I think there is some way.